mozilla_django_oidc/auth.py

```diff
@@ -153,6 +154,15 @@ def retrieve_matching_jwk(self, token):
             raise SuspiciousOperation('Could not find a valid JWKS.')
         return key
 
+    def get_payload_data(self, token, key):
+        """Helper method to get the payload of the JWT token."""
+        if import_from_settings('OIDC_ALLOW_UNSECURED_JWT', False):
```

Comment:
Please add a couple of comments in the code to shortly explain the two cases (token verification and unsecured).

Reply:
Done a couple of lines later.